src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -403,6 +401,9 @@ string relationshipEntity
             _sqlMetadataProvider.Setup(x =>
                 x.ParseSchemaAndDbTableName("TEST_SOURCE_LINK")).Returns(("dbo", "TEST_SOURCE_LINK"));
 
+            string discard;
+            _sqlMetadataProvider.Setup(x => x.TryGetBackingColumn(It.IsAny<string>(), It.IsAny<string>(), out discard)).Returns(true);
+
```

Comment:
assuming you need `discard` because `_` isn't valid syntax for Moq?

Reply:
Right, I can't put it directly into the MOQ call.